WolvenKit.App/ViewModels/HomePage/Pages/ModsViewModel.cs

```diff
@@ -0,0 +1,279 @@
+                        break;
+                }
+
+                return false;
+            }
+
+            var result = false;
+            // compile with redmod
+            var redmodPath = Path.Combine(_settings.GetRED4GameRootDir(), "tools", "redmod", "bin", "redmod.exe");
+            if (!File.Exists(redmodPath))
+            {
+                _logger.Error("RedMod tools are not installed. Please go to WolvenKit plugins and install RedMod.");
+                result = await Task.FromResult(false);
+            }
+            else
+            {
+
+
+                var enabledMods = GetEnabledMods().ToList();
+                if (enabledMods.Any())
+                {
+                    IsProcessing = true;
+
+                    var args = $"deploy -root=\"{_settings.GetRED4GameRootDir()}\"";
+
+                    var modsStr = string.Join(' ', enabledMods.Select(x => $"\"{x.Folder}\""));
+                    args += $" -mod={modsStr}";
+
+                    _logger.Info($"WorkDir: {redmodPath}");
+                    _logger.Info($"Running commandlet: {args}");
+                    result = await ProcessUtil.RunProcessAsync(redmodPath, args);
+
+                    IsProcessing = false;
+
+                    if (!result)
+                    {
+                        await Interactions.ShowMessageBoxAsync(
+                            "RedMod deploy failed. Please check the log for details.",
+                            "RedMod",
+                            WMessageBoxButtons.Ok,
+                            WMessageBoxImage.Error);
+                    }
+                    else
+                    {
+                        await Interactions.ShowMessageBoxAsync(
+                            $"Deployed {enabledMods.Count} enabled mods with RedMod.",
+                            "RedMod deploy",
+                            WMessageBoxButtons.Ok,
+                            WMessageBoxImage.Exclamation);
+                    }
+                }
+                else
+                {
+                    _logger.Warning("No mods enabled.");
+                }
+            }
+
+            return result;
+        }
+
+        public ICommand RefreshCommand { get; private set; }
+        private void Refresh() => LoadMods();
+
+        public ICommand OpenModFolderCommand { get; private set; }
+        private void OpenModFolder() => Commonfunctions.ShowFolderInExplorer(_settings.GetRED4GameModDir());
+
+        public ICommand LaunchGameCommand { get; private set; }
+        private void LaunchGame()
+        {
+            try
+            {
+                Process.Start(new ProcessStartInfo
+                {
+                    FileName = _settings.GetRED4GameLaunchCommand(),
+                    Arguments = _settings.GetRED4GameLaunchOptions() ?? "-modded",
+                    ErrorDialog = true,
+                    UseShellExecute = true,
```

Comment:
Is this necessary? I would try to avoid doing this. Since we already know where the executable is, I think it is safe to remove it.

Reply:
remove what exactly?

Reply:
`UseShellExecute`. I don't know why the line is not highlighted or something like that.

Reply:
we can try :D